src/sass/modules/_m-footer.scss

```diff
@@ -19,7 +19,94 @@
   }
 }
 
-  .footer-inner {
+.feedback-widget {
+  padding: 1.5em 0;
+  background-color: $color-gray-lightest;
+  color: $color-gray-dark;
+
+  .row {
+    padding: 0 0.9375rem;;
+  }
+
+  .feedback-widget-title {
```

Comment:
Do these need to be nested? You've prefixed the class names, so it seems unnecessary.

Reply:
No I just like the style of having it all contained like that. I prefixed the classes just to be explicit. Should I try to flatten this?

Reply:
I think avoiding nesting makes sass easier to maintain, so that would be my recommendation.

Reply:
I guess I usually look at in inversely, in that I try to keep object state as contained as possible and only let it propagate throughout Redux state if there's a clear incentive to do so.